src/main/java/gregtech/common/blocks/BlockFrame.java

```diff
@@ -310,18 +308,18 @@ public BlockFaceShape getBlockFaceShape(@Nonnull IBlockAccess worldIn, @Nonnull
     @Override
     @SideOnly(Side.CLIENT)
     public void onTextureStitch(TextureStitchEvent.Pre event) {
-        for (IBlockState state : this.getBlockState().getValidStates()) {
-            Material material = state.getValue(variantProperty);
-            event.getMap().registerSprite(MaterialIconType.frameGt.getBlockTexturePath(material.getMaterialIconSet()));
-        }
```

Comment:
Is this method intentionally left empty?

Reply:
It still needs declaration because it's abstract.